pkg/operator/sync.go

```diff
@@ -283,12 +284,16 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		if err != nil {
 			klog.Warningf("could not find configmap specified in image.config.openshift.io/cluster with the name %s", cfg.Spec.AdditionalTrustedCA.Name)
 		} else {
-			for key, d := range cm.Data {
-				raw, err := base64.StdEncoding.DecodeString(d)
+			newKeys := sets.StringKeySet(cm.Data).List()
+			sort.Strings(newKeys)
```

Comment:
suggestion: Delete the `sort.Strings()` calls. The `.List()` method attached to the Set object will sort the list before returning it. By comparison, Set objects also have an `.UnsortedList()` method that returns the values unsorted.

Reply:
done